packages/renderer/src/extract-audio.ts

```diff
@@ -0,0 +1,22 @@
+import {startLongRunningCompositor} from './compositor/compositor';
+import type {VideoMetadata} from './compositor/payloads';
+import type {LogLevel} from './log-level';
+
+export const extractAudio = async (options: {
+	videoSource: string;
+	audioOutput: string;
+	logLevel?: LogLevel;
+}): Promise<VideoMetadata> => {
+	const compositor = startLongRunningCompositor({
+		maximumFrameCacheItemsInBytes: null,
+		logLevel: options?.logLevel ?? 'info',
+		indent: false,
+	});
+	const metadataResponse = await compositor.executeCommand('ExtractAudio', {
+		input_path: options.videoSource,
+		output_path: options.audioOutput,
+	});
+	compositor.finishCommands();
+	await compositor.waitForDone();
+	return JSON.parse(metadataResponse.toString('utf-8')) as VideoMetadata;
```

Comment:
@JonnyBurger why did u add a metadata response? extract-audio just returns Ok(vec![])

Reply:
i just realize i didn't push the ts wrapper, sorry my bad but i did almost exactly the same

Reply:
Oh yeah, that's why I did push it, I forgot to rename it here.

Thanks for going back to this PR, looking mostly good though!